portal/models/intervention_strategies.py

```diff
@@ -182,7 +182,7 @@ def update_user_card_html(intervention, user):
             """.format(user.display_name)
             link_label = 'Begin questionnaire'
             link_url = url_for('assessment_engine_api.present_assessment',
-                               instrument_id='epic26')
+                               instrument_ids='epic26')
```

Comment:
If you're expecting an array, shouldn't this be a "list" of instruments?  i.e.
`instrument_ids=['epic26']`